src/components/FlowChart/Sidebar.jsx

```diff
@@ -343,6 +352,7 @@ class FlowChartSidebar extends React.Component {
             ? this.renderEditLink(links[selected.id], nodes)
             : this.renderEditNode(nodes[selected.id])
           : this.renderPodMenu()}
+          {readonly&&this.renderReadOnly()}
```

Comment:
you have to jump to the code on top to understand what happens. You can't read in one flow.